teamfoundationcontrol.cpp

```diff
@@ -105,14 +109,8 @@ bool TeamFoundationControl::managesDirectory(const QString &directory, QString *
     if (!isManaged)
         return false;
 
-    QDir dir(directory);
-    while (true)
-    {
-        QTC_ASSERT(topLevel, return true);
-        *topLevel = dir.absolutePath();
-        if (!dir.cdUp() || !managesDirectoryEx(dir.absolutePath()))
-            break;
-    }
+    QTC_ASSERT(topLevel, return true);

```

Comment:
Well it looks like QTC only ever calls this with the project directory. 
As stated in the documentation QTC then assumes all directories underneath the project dir are managed by the same VCS.

I suppose one could have wonky setups where the project resides in a directory that isn't actually equal to the top level directory.

Maybe make the behavior configurable?